listenbrainz/webserver/static/js/src/SoundcloudPlayer.tsx

```diff
@@ -58,7 +58,6 @@ export default class SoundcloudPlayer
   implements DataSourceType {
   iFrameRef?: React.RefObject<HTMLIFrameElement>;
   soundcloudPlayer?: SoundCloudHTML5Widget;
-  supportsSearch = false;
```

Comment:
This is probably leftover from trying something wy back when, was never used AFAIK and was never cleaned up.